src/components/profileMenu/ProfileMenu.tsx

```diff
@@ -0,0 +1,107 @@
+import { Menu, rem, ActionIcon } from '@mantine/core';
+import {
+	IconUserFilled,
+	IconLogout,
+	IconUser,
+	IconMapPins,
+	IconLogin,
+	IconHistory,
+} from '@tabler/icons-react';
+
+interface ProfileMenuProps {
+	isLogged: boolean;
+}
+
+const ProfileMenu = (props: ProfileMenuProps) => {
+	const handleAccount = () => {
+		console.log('Mon compte');
+	};
+
+	const handleTrips = () => {
+		console.log('Mes voyages');
+	};
+
+	const handleLogout = () => {
+		console.log('Se déconnecter');
+	};
+
+	return (
```

Comment:
even though the functionality isn't there yet, I'm not a fan of the console.log in the code.

Reply:
je pense que c'est pour implémenter a la suite ce qu'il y aura dans les fonctions donc why not